src/Mod/Sketcher/App/SketchAnalysis.cpp

```diff
@@ -87,6 +87,15 @@ struct SketchAnalysis::Vertex_Less
     double tolerance;
 };
 
+struct SketchAnalysis::VertexID_Less
+{
+    bool operator()(const VertexIds& x,
+                    const VertexIds& y) const
+    {
+        return (x.GeoId < y.GeoId || ((x.GeoId == y.GeoId) && (x.PosId < y. PosId)));
```

Comment:
Isn't PosId now an enum class?

Reply:
@AjinkyaDahale yes it is

Reply:
Not a problem. 'LESS' comparing functions should just ensure a transitive ordering in all cases, and that when 2 equal objects are compared, it returns false in both argument orders. ;)